WalletWasabi/WabiSabi/Models/Serialization/OwnershipProofJsonConverter.cs

```diff
@@ -3,24 +3,21 @@
 using System;
 using WalletWasabi.Crypto;
 
-namespace WalletWasabi.JsonConverters
+namespace WalletWasabi.WabiSabi.Models.Serialization
 {
 	public class OwnershipProofJsonConverter : JsonConverter
 	{
-		/// <inheritdoc />
```

Comment:
Not like I care, but why were these removed? 😄

Reply:
I think it is pure noise. It doesn't add anything of value, it is something that is there without good reason.